.github/workflows/ci.yml

```diff
@@ -40,7 +40,7 @@ jobs:
       - name: Check imports using isort
         run: isort --check-only ${{ matrix.executor-path }}
       - name: Black check formatting
-        run: black --check ${{ matrix.executor-path }}
+        run: black --check --exclude='audio_clip/|vggish/' ${{ matrix.executor-path }}
```

Comment:
We should already have this in `pyproject.toml`?

Reply:
It is, but it's not used in CI

Reply:
That's strange, because it should be. Black is executed from the root, so it should be able to read these settings

Reply:
Idk, it failed before I added this. Opened a ticket https://github.com/jina-ai/executors/issues/306